examples/memnn/babi.py

```diff
@@ -0,0 +1,66 @@
+class Query(object):
+
+    def __init__(self, sentence, answer, fact):
+        self.sentence = sentence
+        self.answer = answer
+        self.fact = fact
+
+
+class Sentence(object):
+
+    def __init__(self, sentence):
+        self.sentence = sentence
+
+
+def split(sentence):
+    return sentence.lower().replace('.', '').replace('?', '').split()
+
+
+def convert(vocab, words):
+    return [vocab[w] for w in words]
+
+
+def parse_line(vocab, line):
```

Comment:
I think it's great if a short summary of the dataset format is given somewhere in the example.